DynamicTablesPkg/Include/ArmNameSpaceObjects.h

```diff
@@ -420,6 +420,48 @@ typedef struct CmArmGenericWatchdogInfo {
   UINT32    Flags;
 } CM_ARM_GENERIC_WATCHDOG_INFO;
 
+/** The ARM_PCI_OSC_INFO union/struct describes the features that FW will
+ * grant native control over via _OSC method.
+*/
+typedef union PciOscControlBufferInfo {
+  struct {
+    /// Allow OS Native control of Hot Plug if set
+    UINT32     AllowNativeHotPlugControl : 1;
+
+    /// Allow Native SHPC Control
+    UINT32     AllowNativeShpcControl : 1;
+
+    /// Allow Native Power Management Events Control
+    UINT32     AllowNativePmeControl : 1;
+
+    /// Allow Native Advanced Error Reporting Control
+    UINT32     AllowNativeAerControl : 1;
+
+    /// Allow Native Capability Structure Control
+    UINT32     AllowNativeCapabilityControl : 1;
+
+    /// Allow Native Latency Tolerance Reporting Control
+    UINT32     AllowNativeLtrControl : 1;
+
+    /// Firmware sets this bit to 1 to indicate that it 
+    /// will suppress error notification caused by surprise hot remove
+    /// (Completion Time Out errors)
+    UINT32     SupressSurpriseHotRemoveErrors : 1;
+
+    /// Allow Native Downstream Port Containment Control
+    UINT32     AllowNativeDpcControl : 1;
+
+    /// Allow Native Completion Timeout Control
+    UINT32     AllowNativeCompletionTimeoutControl : 1;
+
+    /// Allow Native System Firmware Intermediary Configuration Control
+    UINT32     AllowNativeSfiControl : 1;
+
+    UINT32     Reserved22 : 22;
+  };
+  UINT32     AsUint32;
+} ARM_PCI_OSC_CONTROL_BUFFER_INFO;

```

Comment:
Can we please add `MU_CHANGE` tags around the updated code blocks? It will help the integration process next time we pick up the next release tag. Thanks!